src/stories/PineTypes.js

```diff
@@ -1,81 +0,0 @@
-import { h, Component } from 'preact'
```

Comment:
I wouldn't expect to see such a deletion in this PR.

Reply:
Yeah it got commited by mistake I think